firefox-ios/firefox-ios-tests/Tests/L10nSnapshotTests/L10nBaseSnapshotTests.swift

```diff
@@ -20,7 +20,7 @@ class L10nBaseSnapshotTests: XCTestCase {
                 LaunchArguments.SkipContextualHints,
                 LaunchArguments.DisableAnimations]
 
-    override func setUp() {
+    @MainActor override func setUp() {
```

Comment:
With these changes tests are running now

Reply:
🎉 All L10n tests pass on my end. Let me run the tests on Bitrise.

Reply:
On Bitrise, the following test(s) failed: `testFakespotAvailable`.
https://app.bitrise.io/build/7d2462c3-4ee8-4e2e-b759-4ad5f6b2ad97

Fakespot tests could be related to network conditions. Could we still merge this work?

Reply:
That's an intermittent failure, we can land this (with the changes commented required) and try to fix that test / make it more robust